projects/vdk-plugins/vdk-duckdb/setup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import setuptools
+
+"""
+Builds a package with the help of setuptools in order for this package to be imported in other projects
+"""
+
+__version__ = "0.1.0"
+
+setuptools.setup(
+    name="vdk-duckdb",
+    version=__version__,
+    url="https://github.com/vmware/versatile-data-kit",
+    description="Simple description of my project.",
```

Comment:
There should be a short description here.

Reply:
Thank you for the review and tips. Will change the things you pointed out.